cmd/jaeger/config-query.yaml

```diff
@@ -0,0 +1,41 @@
+service:
+  extensions: [jaeger_storage, jaeger_query, healthcheckv2]
+  pipelines:
+    traces:
+      receivers: [nop]
+      processors: [batch]
+      exporters: [nop]
+  telemetry:
+    resource:
+      service.name: jaeger-query
+    metrics:
+      level: detailed
+      address: 0.0.0.0:8889
```

Comment:
```suggestion
      readers:
        - pull:
            exporter:
              prometheus:
                host: 0.0.0.0
                # use different port to avoid conflict with collector
                port: 8889
```